Revoke-Obfuscation.psm1

```diff
@@ -1178,6 +1185,31 @@ http://www.leeholmes.com/blog/
                 }
             }
         }
+        "Helix" {
+            # Use API to pull eventid=4104.
+            # Defeat the 1k event limitation using the "stream"="true" option in your request.
+            # Stream returns raw JSON which needs to be split by "`n" and convertfrom-json before providing to this function
+            
+            # Parse out scriptblock text ID#
+            $Helix.msg -match "\((\d+)\sof\s(\d+)\)"|Out-Null
+            
+            # Perform renaming so that structure matches that of [System.Diagnostics.Eventing.Reader.EventLogRecord] objects.
+            [Object[]] $EventLogRecord = $Helix | Select-Object `
+                @{ Name = 'id'              ; Expression = { [int]$_.eventid } },
+                @{ Name = 'TimeCreated'     ; Expression = { [datetime]$_.eventtime } },
+                @{ Name = 'LevelDisplayName'; Expression = { $_.severity } },
+                @{ Name = 'HostName'        ; Expression = { $_.hostname } },
+                @{ Name = 'Properties'      ; Expression = { `
+                    @(
```

Comment:
The properties should be:
P0: Sequence number
P1: Total in sequence
P2: Content
P3: Script block ID
P4: Path

Does "Info" hold the content?
I think $_.ProcessId is wrong